components/browser/menu/src/main/java/mozilla/components/browser/menu/WebExtensionBrowserMenuBuilder.kt

```diff
@@ -33,7 +33,7 @@ class WebExtensionBrowserMenuBuilder(
     private val store: BrowserStore,
     private val webExtIconTintColorResource: Int = NO_ID,
     private val onAddonsManagerTapped: () -> Unit = {},
-    private val appendExtensionActionAtStart: Boolean = false
+    private val appendExtensionSubMenuAtStart: Boolean = false
```

Comment:
@psymoon This is breaking change on my side. Should we change that variable name in fenix?
https://github.com/mozilla-mobile/fenix/blob/3c2c7f705cb03895dcb090ae98fa9718cdeeef69/app/src/main/java/org/mozilla/fenix/components/toolbar/DefaultToolbarMenu.kt#L73

Reply:
https://github.com/mozilla-mobile/fenix/pull/11979